style_editor.html

```diff
@@ -254,7 +254,7 @@
 
 /* Create an active/current tablink class */
 .tab button.active {
-  background-color: #ccc;
+  background-color: #6ADA;
```

Comment:
Is this a transparent color?

Reply:
Would you prefer #A1D1F3?

Reply:
I'm asking because 6ADA has four characters.
I'm used to colors having three or six characters, I'm not sure what kind of color #6ADA is...

Reply:
I made it #A1D1F3 instead across all the CSS that used #6ADA